wolfSSL/src/chapter02.md

```diff
@@ -628,6 +1002,86 @@ Enables Blake2s algorithm support
 
 Enables Signaling Cipher Suite Value(SCSV) support on the server side. This handles the cipher suite 0x56 0x00 sent from a client to signal that no downgrade of TLS version should be allowed.
 
+#### HAVE_AEAD
+
+Implements the use of AEAD and is required for TLS 1.3.
+
+#### HAVE_AES_CBC
+
+Enable option for AES.
+
+#### HAVE_AES_DECRYPT
+
+This setting controls whether the decrypt direction of the various AES modes is supported.
+
+#### HAVE_ALPN
+
+Crypto enables the option for application-layer protocol negotiation.
+
+#### HAVE_CAVIUM_OCTEON_SYNC
+
+By default, the OCTEON's global variables are all thread local. This tag allows them to be shared between threads.
```

Comment:
This enables the blocking (synchronous) version of the Marvell Cavium/Octeon hardware. The asynchronous version is in the wolfAsyncCrypt repo here https://github.com/wolfSSL/wolfAsyncCrypt/tree/master/wolfcrypt/src/port/cavium